synapse/rest/client/v2_alpha/sync.py

```diff
@@ -236,6 +236,7 @@ async def encode_response(self, time_now, sync_result, access_token_id, filter):
                 "leave": sync_result.groups.leave,
             },
             "device_one_time_keys_count": sync_result.device_one_time_keys_count,
+            "org.matrix.msc2732.device_unused_fallback_key_types": sync_result.device_unused_fallback_key_types,
```

Comment:
Does this mean that when this is standardized we'll need to provide this data as both `org.matrix.msc2732.device_unused_fallback_key_types` and `device_unused_fallback_key_types`? That's unfortunate bloat to the sync message.

Reply:
Yeah, that's how unstable prefixes would work in general.  This would only be for a little while, though.  After some time, we could drop `org.matrix.msc2732.device_unused_fallback_key_types`.